### DISCUSSION TITLE
disable lsp hover "No information available"  in snacks notifier

### DISCUSSION THREAD

**Discussion by bibjaw99** [folke/snacks.nvim#408] [Q&A]
I used to have this chunk in my config for disabling the message while using nvim-notify. I want to do something similar with snacks if it is possible. 
```
local banned_messages = { "No information available" }
vim.notify = function(msg, ...)
	for _, banned in ipairs(banned_messages) do
		if msg == banned then
			return
		end
	end
	return require("notify")(msg, ...)
end
```
![2024-12-29_23-57-32](https://github.com/user-attachments/assets/431974cb-835d-44c0-94fa-8b17f4165f81)

**Comment by ejsadiarin** [comment]
@bibjaw99  you can configure the filter function in the opts.notifier, it accepts snacks.notifier.Notif opts as param that you can use to access the notifications generated by the notifier.

this is in my config:
```lua
...
            notifier = {
                enabled = true,
                timeout = 3000,
                style = "fancy",
                filter = function(n)
                    if n.msg == "No information available" then
                        return false
                    end
                    return true
                end,
            },
...
```

you can expand on that snippet with a `filtered_msgs = {"..."}` and loop through those strings that you want filtered (or use a map for better performance)
